Configuration/TypoScript/setup.typoscript

```diff
@@ -223,14 +223,11 @@ page {
                     }
                 }
             }
-            40 = TYPO3\CMS\Frontend\DataProcessing\MenuProcessor
+            40 = TYPO3\CMS\Frontend\DataProcessing\LanguageMenuProcessor
             40 {
-                special = language
+                #special = language
```

Comment:
No useless comments please

Reply:
it's work in progress, so cleanup follows at the end